packages/ethereum/ethereum-light-client/src/verify.rs

```diff
@@ -243,7 +244,7 @@ pub fn validate_light_client_update<V: BlsVerify>(
         finalized_root,
         update.finality_branch.into(),
         FINALITY_BRANCH_DEPTH,
-        get_subtree_index(FINALIZED_ROOT_INDEX),
+        get_subtree_index(FINALIZED_ROOT_GINDEX_ELECTRA),
```

Comment:
Can we do all of this [similar to the spec](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/light-client/sync-protocol.md#finalized_root_gindex_at_slot) where this is now a function called `finalized_root_gindex_at_slot`. Then, we return the electra gindex if the slot is past the electra epoch, and otherwise we error.

I feel pretty strong about this, since it makes our code more similar to the specs and way less error prone.